diamm/serializers/website/stats.py

```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```

Comment:
This is redundant. The `label` property is only useful if its value is different from the field name. Think "type =" but you want the serialized field to be "@type" (which is an illegal field name in Python).